pyproject.toml

```diff
@@ -95,10 +95,10 @@ pika = "~1.1.0"
 jsonschema = "~3.2"
 packaging = ">=20.0,<21.0"
 pytz = ">=2019.1,<2021.0"
-rasa-sdk = "^2.0.0a4"
+rasa-sdk = "^2.0.0a5"
```

Comment:
why do we need to change that?

Reply:
I updated sanic in the latest version of rasa-sdk, and the new version of sanic has better Windows support. We had a discussion about it while you were on vacation :)

Reply:
I saw it but wouldn't it automatically pick the latest sdk version and latest sanic version (without changing the minimal alpha version)?